### PR TITLE
Explorer Expansion: New Horizons

### PR DESCRIPTION
**Pull request by Mechoid**
Large amount of work; Animal taming, shearing. Random animal spawns on sif grass/forest turfs. New PoIs, thermal poncho.

Additions:
  - rscadd: "Adds more interactions with animals, like shearing and taming."
  - rscadd: "Sif moss turfs can now spawn animal groups."
  - rscadd: "New PoIs"
  - rscadd: "Thermal poncho attachment, has minor slowdown, but gives thermal protection to the armor it is attached to."
  - rscadd: "Mercenaries now drop their guns again. Most likely to be broken, however they can be repaired. Examining when adjacent will allow you to inspect the gun for what is needed."
  - rscadd: "Mercenary Snipers now exist. They will telegraph their shots approximately 2 seconds in advance, and can fire at 14 tiles away. Will occasionally drop pieces of their PTRs that survive combat."

Tweaks:
  - tweak: "Clothing can now check attachments for temperature resistance."

Bugfix:
  - bugfix: "MHD Howitzer beam effect actually exists again.
  - bugfix: "Xenoresin ground cover is properly colored again."

### CODE REVIEW

**Comment by Atermonera** [comment]
But can we go to pluto

**Comment by Mechoid** [comment]
Do you want me to do carpentry? I'll fucking do carpentry

**Comment by Atermonera** [comment]
Still two merc/all spawners on DoomP

File: code/game/objects/structures/flora/flora.dm [hunk 1]
```diff
@@ -1,478 +1,493 @@
...
+		return
+
+	..()
+
+/obj/structure/flora/pottedplant/attack_hand(mob/user)
+	if(!stored_item)
+		to_chat(user, "<span class='filter_notice'><b>You see nothing of interest in [src]...</b></span>")
+	else
+		if(do_after(user, 10))
+			to_chat(user, "You find [bicon(stored_item)] [stored_item] in [src]!")
```
Comment (Atermonera) [inline code/game/objects/structures/flora/flora.dm:261]: filter tag

**Pull request merged**